MicrobenchmarkSample/benchmarkable/build.gradle

```diff
@@ -1,31 +1,31 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
+apply plugin: "com.android.library"
```

Comment:
can we convert this file to kts as well?

Reply:
Oops. done